curriculum/challenges/english/25-front-end-development/quiz-css-layout-and-effects/66ed8ff4f45ce3ece4053eb4.md

```diff
@@ -65,63 +65,63 @@ Why does CSS typically avoid hiding content when there is overflow?
 
 #### --distractors--
 
-To enhance page design by hiding irrelevant content
+To maintain the original layout and design.
 
 ---
 
-To save memory and load pages faster
+To prevent performance issues in the browser.
 
 ---
 
-To automatically fix broken layout issues
+To allow for better search engine optimization.
 
 #### --answer--
 
-To prevent data loss and make issues noticeable
+To ensure all content is accessible to users.
 
 ### --question--
 
 #### --text--
 
-How can you handle overflow in CSS to hide content or add scrollbars?
+How can you handle vertical overflow in CSS by hiding content or displaying scrollbars?
 
 #### --distractors--
 
-Use `overflow: visible` to hide content and `overflow: auto` to always display scrollbars
+Use `overflow-y: visible` to hide content and `overflow-y: auto` to always display scrollbars.
 
 ---
 
-Use `overflow: collapse` to hide content and `overflow: expand` to display scrollbars
+Use `overflow-y: collapse` to hide content and `overflow-y: expand` to display scrollbars.
 
 ---
 
-Use `overflow: none` to hide content and `overflow: show` to display scrollbars
+Use `overflow-y: none` to hide content and `overflow-y: show` to display scrollbars.
 
 #### --answer--
 
-Use `overflow: hidden` to hide content and `overflow: scroll` to always display scrollbars
+Use `overflow-y: hidden` to hide content and `overflow-y: scroll` to always display scrollbars.
 
 ### --question--
 
 #### --text--
 
-How can you specify different scrolling behaviors for the x-axis and y-axis in CSS?
+How can you specify different horizontal and vertical scrolling behaviors in CSS?
 
 #### --distractors--
 
-By using `scroll-x` and `scroll-y` properties
+By using the `scroll-x` and `scroll-y` properties.
 
 ---
 
-By using `overflow-x` and `overflow-y` separately in every case
+By specifying the same value for `overflow-x` and `overflow-y`.
 
 ---
 
-By passing two values to the `overflow` property, where the first applies to `overflow-y` and the second to `overflow-x`
+By specifying two values for the `overflow` property, where the first value applies to `overflow-y` and the second value to `overflow-x`.
 
 #### --answer--
 
-By passing two values to the `overflow` property, where the first applies to `overflow-x` and the second to `overflow-y`
+By specifying two values for the `overflow` property, where the first value applies to `overflow-x` and the second value to `overflow-y`.
```

Comment:
I don't actually see anywhere we say that `overflow` can be a shorthand to set both overflow properties. I think we should add that to the review page.